llvm/docs/CMake.rst

```diff
@@ -836,6 +836,12 @@ enabled sub-projects. Nearly all of these variable names begin with
   The full list, as of March 2023, is:
   ``AArch64;AMDGPU;ARM;AVR;BPF;Hexagon;Lanai;LoongArch;Mips;MSP430;NVPTX;PowerPC;RISCV;Sparc;SystemZ;VE;WebAssembly;X86;XCore``
 
+  You can also specify ``host`` or ``Native`` to automatically detect and
+  include the target corresponding to the host machine's architecture, or
+  use ``all`` to include all available targets.
```

Comment:
Note that all is already mentioned above (in the part that github helpfully hides by default).

Reply:
Okay, thanks!